libr/core/cmd_flag.c

```diff
@@ -930,7 +930,30 @@ static int cmd_flag(void *data, const char *input) {
 					eprintf ("Invalid name\n");
 				}
 			} else {
-				eprintf ("Cannot find flag (%s)\n", old);
+				eprintf ("Usage: fr [[old]] [new]\n");
+			}
+		}
+		break;
+	case 'N':
+		if (input[1] == ' ' && input[2]) {
+			RFlagItem *item;
+			char *name = str + 1;
+			char *realname = strchr (name, ' ');
+			if (realname) {
+				*realname = 0;
+				realname++;
+				item = r_flag_get (core->flags, name);
+				if (!item && !strncmp (name, "fcn.", 4)) {
+					item = r_flag_get (core->flags, name+4);
+				}
+			} else {
+				realname = name;
+				item = r_flag_get_i (core->flags, core->offset);
+			}
+			if (item) {
+				r_flag_item_set_realname (item, realname);
+			} else {
+				eprintf ("Usage: fN [[name]] [realname]\n");
 			}
 		}
```

Comment:
wait.. if no arguments, we should show the realname of the flag in the current offset imho (or all of them if more than one)